keyboards/enviousdesign/tkl/readme.md

```diff
@@ -0,0 +1,42 @@
+# EnvKB DELIRIUM
+
+![Image of EnvKB TKL PCB](https://i.imgur.com/a7UaS4mh.png)
+
+A low cost standard TKL design which takes inspiration from the CFTKB Mysterium. All versions are designed for reference Pi Picos.
+
+* Keyboard Maintainer: [Envious-Data](https://github.com/envious-data)
+* Hardware Supported: 
+  * Delirium Prototype (`tkl/rev0`)
+  * Delirium (`tkl/rev1`)
+  * Delirium RGB (`tkl/rgb`)
+* Hardware Availability: [Mechboards](https://mechboards.co.uk/products/envkb-delirium-tkl-kit), [GitHub](https://github.com/Envious-Data/Env-KB)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make enviousdesign/tkl/rgb:default
+
+Flashing example for this keyboard:
+
+    make enviousdesign/tkl/rgb:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: *Not enabled by default.* Hold down the top left key and plug in the keyboard. This will also clear the emulated EEPROM, so it is a good first step if the keyboard is misbehaving.
```

Comment:
```suggestion
* **Bootmagic reset**: Hold down the top left key and plug in the keyboard. This will also clear the emulated EEPROM, so it is a good first step if the keyboard is misbehaving.
```